src/riak_kv_vnode.erl

```diff
@@ -2656,17 +2649,19 @@ key_epoch_actor(ActorBin, Cntr) ->
     <<ActorBin/binary, Cntr:32/integer>>.
 
 %% @private highest actor is the latest/greatest epoch actor for a
-%% key. It is the actor we want to increment for the current event,
-%% given that we are not starting a new epoch for the key.  Must work
-%% with non-epochal and epochal actors.  The return tuple is
-%% `{ActorId, Epoch, Counter}' where `ActorId' is the highest ID
-%% starting with `ActorBase' that has acted on this key, undefined if
-%% never acted before. `KeyEpoch' is the highest epoch for the
-%% `ActorBase'. `Counter' is the greatest event seen by the `VnodeId'.
+%% key. It is the actor we want to increment for a new event, if we
+%% are not starting a new epoch for the key.  Must work with
+%% non-epochal and epochal actors.  The return tuple is `{ActorId,
+%% Epoch, Counter}' where `ActorId' is the highest ID starting with
+%% `ActorBase' that has acted on this key. `KeyEpoch' is the highest
+%% epoch for the `ActorBase'. `Counter' is the greatest event seen by
+%% the `VnodeId'.
 -spec highest_actor(binary(), riak_object:riak_object()) ->
```

Comment:
Since we are now allowed to pass `undefined` into this function, we should add it to the type spec.